package.json

```diff
@@ -38,7 +38,7 @@
     "lint-source": "eslint --color --ext .js,.jsx src",
     "lint-demo": "eslint --color --ext .js,.jsx demo",
     "lint-docs": "eslint --color --ext .js,.jsx docs",
-    "lint-test": "eslint --color --ext .js,.jsx test",
+    "lint-test": "eslint --color --ext .js,.jsx --rule 'mocha/no-exclusive-tests:2' test",
```

Comment:
I think we went that route so that we would have an `.eslintrc` in each project for in-editor linters to use